src/cmd/dev.go

```diff
@@ -282,7 +292,19 @@ var devLintCmd = &cobra.Command{
 		}
 		defer pkgClient.ClearTempPaths()
 
-		return lint.Validate(cmd.Context(), pkgConfig.CreateOpts)
+		err = lint.Validate(cmd.Context(), pkgConfig.CreateOpts)
+		var lintErr *lint.LintError
+		if errors.As(err, &lintErr) {
+			common.PrintFindings(lintErr)
+			// Do not return an error if the findings are all warnings.
+			if lintErr.OnlyWarnings() {
+				return nil
+			}
+		}
+		if err != nil {
+			return err
+		}
+		return nil
```

Comment:
We should add something here to tell the user the operation was successful

Reply:
I do not agree. We log if something went wrong and exit 1. If all is good we should do nothing.

Reply:
I believe providing assurances to users that their action was completed successfully provides a better UX so they feel confident what they asked for really occurred. That being said I see the argument that an exit 0 is that feedback and is expected with CLI commands and the UNIX style. 

Looking at some examples across different tools `helm lint` and `pylint` both give output. They do both provide additional information, Helm gives the amount of charts linted while pylint gives a score out of 10 and the improvement over the last run. 

`shellcheck`, `yamllint` and `tflint` all give no output on success.

I am going to approve this, after looking at different tools I think no output is reasonable given our case.